src/Components/Loader/Loader.component.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+  width: 20px;
```

Comment:
- [ ] Instead of using `px` use `rem`. In this situation `20px` would be `1.25rem` (20 ÷ 16). In case you're wondering why I prefer `rem` over `px` here is an article: https://medium.com/@julienetienne/pixels-are-dead-faa87cd8c8b9